### PR TITLE
White-space cleanup in 

### PR DESCRIPTION
**Pull request by paulomorgado**
White-space cleanup in  src/PerfView/SupportFiles/UsersGuide.htm

### CODE REVIEW

File: src/PerfView/SupportFiles/UsersGuide.htm [hunk 1]
```diff
@@ -19,7 +19,7 @@
     </style>
 </head>
 <body>
-    <!--  ************************************************************************************* -->
+    <!- - ************************************************************************************* -->
```
Comment (billwert) [inline src/PerfView/SupportFiles/UsersGuide.htm]: This seems unintentional as it breaks the comment?

Comment (paulomorgado) [reply src/PerfView/SupportFiles/UsersGuide.htm]: OOPS! Fixed!

**Comment by brianrob** [comment]
>             <strong>Response Time Investigations: </strong > By collecting with the 'ThreadTime' option

Typo? </strong > should be </strong>.

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:102 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>             <strong> CPU Investigations:</strong > PerfView has the ability to read the output of the Linux 'Perf Events'

Same here.

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:123 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         the &#39;Back&#39; button to undo any changes yo u made so you can re-select.

Typo: yo u to you.

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:298 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>     you may need to resolve symbols. Unlike managed code , unmanaged

Typo: extra whitespace.

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:330 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         For example, the top line in th e ByName view is

Typo: th e --> the

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:411 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>             Review: what all this time selection, grouping an d folding is for?

an d --> and

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:462 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         <a id="TutorialTopDown">CallTree Vie w (top-down investigations))</a>

Vie w --> View

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:495 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         The following image shows the CallTreeVie w after hitting F7 seven times.

CallTreeVie w --> CallTreeView

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:538 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         dll (this is the Windows OS Kernel ) To dig in more we would first

Extra space right after Kernel.

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:570 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         For example , If we were a developer responsible for the DateTime.get_Now(),

Extra space right after example.

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:581 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         If we go back to the &#39;ByName&#39; view and select the 3792 sample s &#39;Inc&#39;

sample s --> samples

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:586 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         called &#39;GetUtcOffsetFromUniversalTime&#39; and &#39;GetDatePart&#39 ;

Extra space at the end of the line before the ;

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:607 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         you can also do the same thing to dril l into exclusive samples.

dril l --> drill

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:616 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         contains CPU information for ALL processes in the system , however most analyses

Extra space after system.

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:806 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         CPU samples for all processes, and then use a GroupPat tha t erases the process

tha t --> that

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:893 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
>         Even with 1000s of samples , there is still &#39;noise&#39; that is at least in the 3% range (sqrt(1000) ~= 30 = 3%). This error gets larger as the methods / groups being investigated

Extra whitespace after samples but before the comma.

---
Refers to: src/PerfView/SupportFiles/UsersGuide.htm:898 in c679343. [](commit_id = c6793435ff6d299ef74a02daba2f24c4f34bd2f2, deletion_comment = False)

**Comment by brianrob** [comment]
@paulomorgado, I started reviewing this, but it seems that there are a bunch of extra whitespaces introduced by this as well.  Before I go too much further, does the UsersGuide display incorrectly for you without this change?

**Comment by paulomorgado** [comment]
That's odd!

I intended only to replace `&nbsp;` with ` ` and consolidate multiple ` `s.

I guess that was one of those times where regular expressions solve problems you wouldn't have if you hadn't messed with regular expressions in the first place. :smile:

**Pull request closed without merging**